tests/binary-expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -145,6 +145,37 @@ if (
 
 `;
 
+exports[`inline-jsx.js 1`] = `
+const user = renderedUser || <div><User name={this.state.user.name} age={this.state.user.age} /></div>;
+
+const user = renderedUser || shouldRenderUser && <div><User name={this.state.user.name} age={this.state.user.age} /></div>;
+
+const avatar = hasAvatar && <Gravatar user={author} size={size} />;
+
+const avatar = (hasAvatar || showPlaceholder) && <Gravatar user={author} size={size} />;
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+const user = renderedUser || (
+  <div>
+    <User name={this.state.user.name} age={this.state.user.age} />
+  </div>
+);
+
+const user =
+  renderedUser ||
+  (shouldRenderUser && (
+    <div>
+      <User name={this.state.user.name} age={this.state.user.age} />
+    </div>
+  ));
```

Comment:
At first glance, this double-paren looked kinda weird, but after thinking it over, I think I actually prefer it to any of the alternatives